roles/docker/tasks/main.yml

```diff
@@ -18,6 +18,10 @@
   when: dns_mode != 'none' and resolvconf_mode == 'docker_dns'
   tags: facts
 
+- include: container_storage_setup.yml
+  when: docker_container_storage_setup and ansible_os_family == "RedHat"
```

Comment:
FYI The centos/RHEL docker package includes a variant of container-storage-setup, and this feature mimics that feature.